scripts/portfolio.js

```diff
@@ -1,25 +1,19 @@
-// Old Constructor
-    /*
-    var projectDataItems = [];
 
-    function ProjectItem (object){
-      this.name = object.name;
-      this.id = object.id;
-      this.class = object.class;
-      this.creators = object.creators;
-      this.date = object.date;
-      this.repoUrl = object.repoUrl;
-      this.location = object.location;
-      this.summary = object.summary;
-    };
-    */
+    // var projectDataItems = [];
 
-// New Constructor
-function ProjectItem (objects) {
-  for (key in objects) {
-    this[key] = objects[key];
-  };
-};
+(function (module) {
+
+  function ProjectItem (object){
+    this.name = object.name;
+    this.id = object.id;
+    this.class = object.class;
+    this.developers = object.developers;
+    this.date = object.date;
+    this.occasion = object.occasion;
+    this.link = object.link;
+    this.image = object.image;
+    this.summary = object.summary;
+  }
```

Comment:
I know you mentioned this in your comment.  This will be an easy one to refactor.